### PR TITLE
Update renewing of certs guide

### PR DESCRIPTION
**Pull request by carleton-dds**
[AT-5244](https://ccpo.atlassian.net/browse/AT-5244)

@okaycj-atat, @dandds  and I renewed the certs for staging and prod this week. This is an update to the README.md with new details on how to perform this task.

### CODE REVIEW

File: deploy/README.md [hunk 1]
```diff
@@ -115,28 +122,69 @@ data:
...
+
+- CONTAINER_IMAGE
+- PORT_PREFIX
+- MAIN_DOMAIN
+- AUTH_DOMAIN
+- VMSS_CLIENT_ID
+- KV_NAME
+- TENANT_ID
+- KV_MI_ID
+- KV_MI_CLIENT_ID
```
Comment (dandds) [inline deploy/README.md:139]: Do you want to update and link to the "Applying K8s configuration" section earlier in the README instead?

File: deploy/README.md [hunk 2]
```diff
@@ -79,9 +79,14 @@ First start certbot in manual mode:
...
-(You may need to supply `--config-dir`, `--work-dir`, and `--logs-dir` options; certbot may try to write to directories that require root privileges by default.)
+If certbot tries to write to directories that require root privileges, you can instead run the following command:
+```
+certbot --config-dir ~/.certbot/config --work-dir ~/.certbot/workdir --logs-dir ~/.certbot/logs certonly --manual
+```
+You may be prompted to enter an email address, generally we've used the address of our tech lead in the past. If you asked to subscribe to any newsletters, decline.
+You may also be prompted to agree to a terms of services, which you should do. 
 
-You will be prompted to enter the domain names you want a cert for. Enter **both** the main and auth domains. For instance:
+You will be prompted to enter the domain names you want a cert for. Enter **both** the main and auth domains as a comma seprated list. For instance:
```
[the remark this answers is not in the document]
Comment (carleton-dds) [reply deploy/README.md]: All set: https://github.com/dod-ccpo/atst/pull/1533/commits/6f92bca7f900868108e0507b8f3dd43a076cc4a1

File: deploy/README.md (hunk 1 above)
Comment (carleton-dds) [reply deploy/README.md]: Updated, however there were still two configs I believe weren't listed in the first section
https://github.com/dod-ccpo/atst/pull/1533/commits/b4a368bf74029425e26529b9ced8d188b775dab5

Comment (dandds) [reply deploy/README.md]: Right, I meant add them to that section. We can re-assess the entire doc on a different ticket, though.

Comment (carleton-dds) [reply deploy/README.md]: Oops, I think I misinterpreted your comment

**Pull request merged**